main/test/ca/uwaterloo/flix/language/phase/TestNamer.scala

```diff
@@ -20,6 +20,7 @@ import ca.uwaterloo.flix.TestUtils
 import ca.uwaterloo.flix.language.errors.NameError
 import ca.uwaterloo.flix.util.Options
 import org.scalatest.FunSuite
+import ca.uwaterloo.flix.language.errors.ResolutionError
```

Comment:
If there are resolution errors then these tests should be in `TestResolver`.

Or perhaps something is wrong with the tests?

Reply:
Yes, this is where I originally put them, in fact, but it felt wrong to have tests that only pass because of code that's in the namer.

But I can put them back there if you prefer?

Reply:
Import can be removed :)